mne/epochs.py

```diff
@@ -2095,6 +2096,7 @@ def to_data_frame(self, picks=None, index=None,
         %(df_copy)s
         %(df_longform_epo)s
         %(df_time_format)s
+        %(verbose)s
 
             .. versionadded:: 0.20
```

Comment:
Keep the versionadded with the `df_time_format` param
```suggestion

            .. versionadded:: 0.20
        %(verbose)s

            .. versionadded:: 1.0
```
You could add one for `verbose`, too, but we sometimes don't bother for that `kwarg`